internal/builder/v1beta1/pipeline_test.go

```diff
@@ -152,14 +152,15 @@ func TestPipeline(t *testing.T) {
 			}, {
 				Name: "task-with-taskSpec",
 				TaskSpec: &v1beta1.EmbeddedTask{
-					Metadata: v1beta1.PipelineTaskMetadata{
-						Labels:      map[string]string{"label": "labelvalue"},
-						Annotations: map[string]string{"annotation": "annotationvalue"},
+					EmbeddedSpec: v1beta1.EmbeddedSpec{
+						Metadata: v1beta1.PipelineTaskMetadata{
+							Labels:      map[string]string{"label": "labelvalue"},
```

Comment:
Please note, those already using the PipelineTaskMetadata field in the EmbeddedTask, will have to make this change. 
//cc @sbwsg

Reply:
Ah I see, the problem becomes for users who are interacting with Tekton by importing its types - they now have to update their code to populate the embedded struct.  Is that right?

That's a really interesting point.  I think this is an argument in favor of keeping the two structs (`EmbeddedRunSpec` and `EmbeddedTask`) as totally separate copies in that case.

Reply:
ok, will update the code accordingly.